eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -489,6 +489,48 @@
           "old": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "new": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "justification": "Removing Jackson annotations from Job Router in transition to stream-style."
+        },
+        {
+          "regex": true,
+          "code": "java\\.annotation\\.(attributeRemoved|attributeAdded)",
+          "old": "class com\\.azure\\.resourcemanager\\.cosmos\\.models.*",
+          "justification": "Fix incorrect json attributes."
+        },
+        {
+          "regex": true,
+          "code": "java\\.method\\.removed",
+          "old": "method java\\.lang\\.Integer com\\.azure\\.resourcemanager\\.cosmos\\.models\\.ServiceResourceCreateUpdateParameters::instanceCount\\(\\)",
+          "justification": "Remove getter for Value from ServiceResourceCreateUpdateParameters since it's in the constructor."
```

Comment:
wrong reason

Say "property flatten disabled due to model changed to polymorphic"

Reply:
Fixed in the new version.